docs/source/install-macos.rst

```diff
@@ -0,0 +1,48 @@
+.. _install-macos:
+
+###########################
+Installation - macOS / OS X
+###########################
+
+.. important:: THIS IS THE COZMO SDK BETA. The SDK is under development and is subject to change.
+
+This guide provides instructions on installing the Cozmo SDK for computers running with a macOS operating system.
+
+-------------------
+Python Installation
+-------------------
+
+1. Install `Homebrew <http://brew.sh>`_ on your system according to the latest instructions. If you already had brew installed then update it by opening a Terminal window and typing in the following::
+
+    brew update
+
+2. Once Homebrew is installed and updated, type the following into the Terminal window to install the latest version of Python 3::
+
+    brew install python3
+
+----------------
+SDK Installation
+----------------
+
+To install the SDK, type the following into the Terminal window::
+
+    pip3 install --user `cozmo[camera]`
```

Comment:
@jfhardy misinterpreted my comment - there should be single quotes around the `cozmo[camera]` portion and backticks around the entire command so that Sphinx formats it correctly.